Anytype/Sources/ServiceLayer/AppVersionService/AppVersionService.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+import AnytypeCore
+
+protocol AppVersionServiceProtocol: AnyObject {
+    func prepareData()
+    func newVersionIsAvailable() async throws -> Bool
+}
+
+actor AppVersionService: AppVersionServiceProtocol {
+    
+    private var vesionChecked = false
+    private var needsUpdate = false
+    
+    nonisolated func prepareData() {
+        Task {
+            try await updateData()
+        }
+    }
+    
+    func newVersionIsAvailable() async throws -> Bool {
+        if vesionChecked {
+            return needsUpdate
+        }
+        
+        try await updateData()
+        
+        return needsUpdate
+    }
+    
+    // MARK: - Private
+    
+    private func updateData() async throws {
+        vesionChecked = true
+        
+        guard let appVersion = currentVersion() else { return }
+        
+        guard let url = URL(string: "https://itunes.apple.com/lookup?bundleId=\(AppId.production)") else { return }
```

Comment:
For all builds we get production version

Reply:
Why dont you use #IF Debug for different app ids?